app/models/user.rb

```diff
@@ -4,6 +4,16 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+ acts_as_messageable
+
+ def name
+   username
+ end
+
+ def mailboxer_email(object)
```

Comment:
Unused method argument - `object`. If it's necessary, use `_` or `_object` as an argument name to indicate that it won't be used. You can also write as `mailboxer_email(*)` if you want the method to accept any arguments but don't care about them.<br>Inconsistent indentation detected.